krymskiy_sviatoslav/graph_generator.cpp

```diff
@@ -0,0 +1,66 @@
+#include <cassert>
+#include <iostream>
+#include <vector>
+
+#include "graph_generator.hpp"
+
+constexpr int kVerticesCount = 14;
+
+Graph::VertexId Graph::get_new_vertex_id() const {
+  return current_vertex_id_;
```

Comment:
Чтобы вернуть данные и увеличить счетчик одним действием - можно сделать так
```suggestion
  return current_vertex_id_++;
```